department-of-veterans-affairs/caseflow
# Add generic task ui

Resolves #6703

### Description
Add organizational tasks actions. Note we don't yet hide the actions dropdown when the task is on hold. For a future PR.

### Testing Plan
1. Go to /organization/translation with translation user
1. Assign task to user
1. Go to /queue
1. See task and assign to organization

![genericui](https://user-images.githubusercontent.com/3885236/46482114-f4d54300-c7c2-11e8-8eda-c1c2bd683d76.gif)
